src/Administration/Resources/app/administration/src/module/sw-login/view/sw-login-recovery/index.ts

```diff
@@ -43,7 +41,7 @@ export default Component.wrapComponentConfig({
         sendRecoveryMail() {
             this.$emit('is-loading');
 
-            this.userRecoveryService
+            Shopware.Service('userRecoveryService')
```

Comment:
Do you mind explaining in short what this change does? Are injected services only loaded when the client is authenticated?

Reply:
We have a timing issue in the login which is not easy solvable because of the different boot process for it. Access to this service is therefore only available after the initial rendering